app/controllers/api/locale.php

```diff
@@ -2,8 +2,11 @@
 
 use Utopia\Database\Document;
```

Comment:
Changes looks good, let's just group all `use` statements by namespaces, a-z.

Reply:
Sure , I made the changes @eldadfux